common/spec/dependabot/dependency_group_spec.rb

```diff
@@ -117,17 +117,17 @@
       context "when no dependencies are assigned to the group" do
         it "returns true if the dependency matches a pattern" do
           expect(dependency_group.dependencies).to eq([])
-          expect(dependency_group.contains?(test_dependency1)).to be_truthy
+          expect(dependency_group).to be_contains(test_dependency1)
```

Comment:
`to be_contains` reads a little odd. Does `to include` work?

Same for below.

Reply:
Since this is a `Dependabot::DependencyGroup` we'd want to alias `contains?` to `include?` for it to work, but I agree that this reads weird as is so I'd recommend we make that change :+1:

Reply:
to be honest, I was weirded out by the whole syntax being there at all.  I think I understand the concept, that we want the test to pass only if it is true and not be skipped if it is nil.  Actually, I also see that I somehow got the **.contains?** call away from the **dependency_group** which actually defines the method. I changed it to .to be(true) - or .to be(false) in case there is a 'not' present. Committed change.